templates/playlist/playlist.html

```diff
@@ -6,11 +6,19 @@
 <form method="post" novalidate>
   {% csrf_token %}
   <label>Title:</label>
+  {% if form.title.value is None %}
```

Comment:
this is too hacky. render form in get method, you manually created the form fields which is totally wrong, instead use {{ form.title }} to diplay the form fields